client/analytics/report/orders/table.js

```diff
@@ -103,7 +103,7 @@ class OrdersReportTable extends Component {
 
 	getRowsContent( tableData ) {
 		const { query } = this.props;
-		const currentInterval = getIntervalForQuery( query );
+		const currentInterval = getIntervalForQuery( { interval: 'day', ...query } );
```

Comment:
This doesn't seem to work for me. Maybe this way?
```suggestion
		const currentInterval = getIntervalForQuery( { ...query, interval: 'day' } );
```

Also, this change will have to be made to all other tables that display dates: coupons, customers, downloads and revenue.

Reply:
@Aljullu This seemed to work on all reports for me.  Could you let me know which reports and date range you're seeing it not working?

I don't think we want to swap them since this would always set the interval to `day` and not allow it to be overridden by `query.interval` if set.  If we want it to always show in `mm/dd/yyyy` format and remove the possibility to show hours, I prefer your earlier suggestion of removing `getDateFormatsForInterval()` altogether.

Reply:
I see now.  I was thinking we could still allow hours in this table if we add intraday date ranges, but manually setting to hourly still produces this error.

We could check if the query period is exactly one day and set the interval to `day` and only use `hour` if it's less than 24 hours, but since @LevinMedia noted that there are no current plans to use hours in these tables, I've added the previous solution you mentioned bypassing `getDateFormatsForInterval()`.

I think you were correct in your original review and results in the cleanest approach instead of trying to make `getDateFormatsForInterval()` fit a situation where it isn't needed.